test/proxy_SUITE.erl

```diff
@@ -402,11 +411,20 @@ limit_exceeded(_Config) ->
     ok = libp2p_swarm:stop(ClientSwarm2),
 
     %% check we didn't leak any sockets here
-    ok = test_util:wait_until(fun() ->
-                                      [{_ID, Info}] = ranch:info(),
-                                      0 == proplists:get_value(active_connections, Info) andalso
-                                      0 == proplists:get_value(all_connections, Info)
-                              end),
+    ok = test_util:wait_until(
```

Comment:
Repeating blocks of the same wait_until code? Extract out into it’s own function?